substratevm/src/com.oracle.svm.agent/src/com/oracle/svm/agent/NativeImageAgent.java

```diff
@@ -121,6 +121,7 @@ protected int onLoadCallback(JNIJavaVM vm, JvmtiEnv jvmti, JvmtiEventCallbacks c
         boolean configurationWithOrigins = false;
         int configWritePeriod = -1; // in seconds
         int configWritePeriodInitialDelay = 1; // in seconds
+        boolean trackReflectionMetadata = false;
```

Comment:
Any reason not to enable this by default?

Reply:
Since only the reachable methods are included in the metadata for now, turning this on would include metadata that is not necessarily needed. Is it more important to get running programs from the get go (instead of having to flip this one switch if needed) or is image size in the general case more important here?

Reply:
Enabled by default